Meta/CMake/common_compile_options.cmake

```diff
@@ -101,6 +101,7 @@ elseif (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     # FIXME: This warning seems useful but has too many false positives with GCC 13.
     add_cxx_compile_options(-Wno-dangling-reference)
 elseif (CMAKE_CXX_COMPILER_ID MATCHES "Clang$" AND CMAKE_CXX_SIMULATE_ID MATCHES "MSVC")
+    add_cxx_compile_options(-Wno-implicit-const-int-float-conversion)
```

Comment:
This is regarding the code you mentioned in #browser, right? So you think the warning is not useful (disabled above too)

I'm wondering if it would be better to silence that warning in that `two_pow_bitlength` code I wrote instead.

e.g does casting NumericLimits<MakeUnsigned<T>>::max() to double do it? That code is admittedly a little strange, the noise of a cast doesn't seem too bad in that case.

Reply:
Actually, this warming is disabled in every other config that offers it. It was just missed here. There are many many violations of it, so I think addressing the warning is in scope for an issue or separate PR

Reply:
Ah right, misunderstood the context!